IntegrationAdapters/Program.cs

```diff
@@ -18,7 +18,7 @@ namespace IntegrationAdapters
     public class Program
     {
         public static List<ActionAndBenefitMessage> Messages = new List<ActionAndBenefitMessage>();
-
+        public static String ResponseMessageGrpc;

```

Comment:
use string (basic type) instead of String (class) - same difference as int and Integer in Java